docs/VARIABLES.md

```diff
@@ -2334,7 +2334,7 @@ Default:  "{{rbac_component_additional_system_admins}}"
 
 ### secrets_protection_enabled
 
-Boolean to enable secrets protection on all components except Zookeeper. Starting form CP 7.2.0 Secrets protection. This works only with RBAC.
+Boolean to enable secrets protection on all components except Zookeeper. Starting form CP 7.1.0 Secrets protection. This works only with RBAC.
```

Comment:
```suggestion
Boolean to enable secrets protection on all components except Zookeeper. Starting form CP 7.2.0 Secrets protection. This works only with RBAC.
```

Reply:
Likewise. Coming from docs script. I think we need to fix this variable description coz it'll keep getting bumped up when it shouldn't
I think the var desc wants to say 
`Starting from cp 71x secrets protection, this will only with rbac` but it's not clear from what's currently written. WDYT?